readme.md

```diff
@@ -91,6 +96,20 @@ Default: `false`
 
 Use full-length units: `5h 1m 45s` → `5 hours 1 minute 45 seconds`
 
+#### separateMs
+
+Type: `boolean`<br>
+Default: `false`
+
+Show milliseconds separately, this means they won't be included in the decimal part of the seconds
```

Comment:
Can you include a short code example of before/after using this option with the output? Same with `formatSubMs`.